src/win/internal.h

```diff
@@ -166,6 +166,25 @@ void uv_process_udp_send_req(uv_loop_t* loop, uv_udp_t* handle,
 void uv_udp_close(uv_loop_t* loop, uv_udp_t* handle);
 void uv_udp_endgame(uv_loop_t* loop, uv_udp_t* handle);
 
+/*
+ * DEVICE
+ */
+int uv_device_read_start(uv_device_t* handle, uv_alloc_cb alloc_cb,
```

Comment:
That keep same style function definition like other functions in same file. If changed will break style internal.h.